autonomy/data/Dockerfiles/dev/Dockerfile

```diff
@@ -21,9 +21,6 @@ RUN python -m pip install --force-reinstall pipenv virtualenv --user
 RUN python -m pip uninstall -y setuptools
 WORKDIR /home/ubuntu
 
-RUN aea init --reset --local --ipfs --author ${AUTHOR}
```

Comment:
As indicated [here](https://github.com/valory-xyz/open-autonomy/pull/1139/files#r933792473) I am not sure why this is not required...but it currently breaks the tests.